packages/client/src/contents/types/contents.types.ts

```diff
@@ -6,7 +6,8 @@ export type Targets = {
   country?: string;
   benefits?: string;
   channel?: string;
-} & Record<string, string | undefined>;
+  segments?: string[];
```

Comment:
If segments can have at most 10 elements, what do you think of defining this type as a tuple with conditional indices, like this:

```js
segments?: [string, string?, string?, string?, string?, string?, string?, string?, string?, string?]
```

Reply:
(I am assuming that the first position will always exist if the `segments` property exists in the payload but feel free to change it to optional as well if that is not the case)

Reply:
I understand your suggest, but this type wouldn’t be too long?

Reply:
That is not a problem if it is correct and helps users catch errors. Is it true that it will only have at most 10 segments? If it is, then do not have a problem with the type being too long.

Reply:
You can also define a new type alias for that and use that instead like this:

```js
export type TargetSegments = [string, string?, string?, string?, string?, string?, string?, string?, string?, string?];
...
segments?: TargetSegments;
```

Reply:
In conversation with Ricardo, they decided that the segments should be passed as a string where the segments are seperated by commas